### DISCUSSION TITLE
Next.js+MDX手撸一个开源周刊模板 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#4] [General]
# Next.js+MDX手撸一个开源周刊模板 | J实验室

全栈工程师，Next.js开源手艺人，掘金签约作者，折腾才有未来

https://weijunext.com/article/ship-weekly-newsletter-fast

**Comment by Ceneses** [comment]
整个项目做的不错！

**Comment by jon1301** [comment]
很棒，学习了

**Comment by zawa8** [comment]
нscii font (no unicode) for korea/indiàn làńguàges 
[fontpicker](https://github.com/zava810/weijunext/discussions/1)
